tests/data/small_sky_source_margin/properties

```diff
@@ -1,15 +1,15 @@
 #HATS catalog
 obs_collection=small_sky_source_margin
 dataproduct_type=margin
-hats_nrows=395
+hats_nrows=1717
```

Comment:
Large increase in the number of rows, here and throughout. Increased size of small sky?

Reply:
In the case of margins, it's mostly due to sweeping changes to margin generation made in November that removes the fine filtering in generation, and instead only performs a coarse healpix tile based filtering.